Userland/Services/WebContent/WebContentClient.ipc

```diff
@@ -59,6 +59,7 @@ endpoint WebContentClient
     did_request_minimize_window() => (Gfx::IntRect window_rect)
     did_request_fullscreen_window() => (Gfx::IntRect window_rect)
     did_request_file(DeprecatedString path, i32 request_id) =|
+    did_request_color_picker() =|
```

Comment:
Would be nice to pass in the current colour here so the picker doesn't start at `#ffffffff` every time.

Reply:
It should also initialize to a valid initial value, I think it should be calling reset_algortihm on initialization